spotify/main.py

```diff
@@ -52,6 +52,11 @@ def load_file_or_env(var, default=None):
 # Load from environment variable if set, otherwise default to 0
 STORE_DATA = os.environ.get("STORE_DATA", "False").lower() in ("true", "1", "t")
 
+# Start- and end-date for the data we want to fetch
+# Load from environment variable if set, otherwise default to current date
+START_DATE = os.environ.get("START_DATE", (dt.datetime.now() - dt.timedelta(days=3)).strftime("%Y-%m-%d"))
+END_DATE = os.environ.get("END_DATE", dt.datetime.now().strftime("%Y-%m-%d"))
```

Comment:
maybe we should set the end date to yesterday by default? and start -4 days then as we do not have data today anyways

Reply:
True, good idea. Let me change that.